### ISSUE TITLE
[Bug]: Invalid saturation light with XY color

### ISSUE THREAD

**Issue by MrEbbinghaus** [t0bst4r/matterbridge-home-assistant#142]
### What happened?

There seems to be a bug in the colour conversion. 
It produces an out of spec saturation value for a light that is pure red.

Here is the state of the light.
```yaml
min_color_temp_kelvin: 2000
max_color_temp_kelvin: 6535
min_mireds: 153
max_mireds: 500
supported_color_modes:
  - color_temp
  - xy
color_mode: xy
brightness: 255
color_temp_kelvin: null
color_temp: null
hs_color:
  - 359.765
  - 100
rgb_color:
  - 255
  - 0
  - 0
xy_color:
  - 0.7007
  - 0.299
mode: normal
dynamics: none
supported_features: 40
```

This error prevents the add-on from starting up.

### Your configuration

included: light

### Relevant log output

```shell
ValidationError: Validation error for attribute "currentHue": (Validation/135) Invalid value: 254.83333333333334 is above the maximum, 254.
    at TlvNumberSchema.validateBoundaries (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/tlv/TlvNumber.js:58:13)
    at TlvNumberSchema.validate (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/tlv/TlvNumber.js:83:10)
    at AttributeServer.validateWithSchema (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/cluster/server/AttributeServer.js:119:19)
    at AttributeServer.validator (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/cluster/server/AttributeServer.js:267:12)
    at AttributeServer.processSet (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/cluster/server/AttributeServer.js:326:10)
    at AttributeServer.setLocal (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/cluster/server/AttributeServer.js:319:10)
    at ClusterServer.result.<computed> [as setCurrentHueAttribute] (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/cluster/server/ClusterServer.js:249:98)
    at ColorControlAspect.update (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/devices/aspects/color-control-aspect.js:46:37)
    at LightDevice.updateState (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/devices/home-assistant-device.js:21:26)
    at async HomeAssistantMatterAdapter.create (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/home-assistant/home-assistant-matter-adapter.js:91:9) {
  code: 135,
  clusterCode: undefined,
  fieldName: undefined
}
```

### Version

1.5.1-72

**Comment by t0bst4r** [comment]
fixed it with 1.5.2 - please test and provide feedback 👍

**Comment by MrEbbinghaus** [comment]
Will do. 

While we are at it. Do you know why the max value in Matter is $2^n-1$?
Is 0xFF… used as the value for `null`?
